pkg/ocmagenthandler/ocmagenthandler_deployment.go

```diff
@@ -155,6 +156,7 @@ func buildOCMAgentDeployment(ocmAgent ocmagentv1alpha1.OcmAgent) appsv1.Deployme
 								Weight: 1,
 							}},
 						},
+						PodAntiAffinity: affinity.CreatePodAntiAffinity(),
```

Comment:
Do we need a separated function for this? And there is no parameter needed for this function

Reply:
Yeah, I was thinking to follow the similar pattern as namespace.go in util to decouple this anti-pod-affinity function. It might be more readable to just put the anti-pod-affinity setting here directly.